dropwizard-logging/src/main/java/io/dropwizard/logging/DefaultLoggingFactory.java

```diff
@@ -176,4 +177,15 @@ private Logger configureLevels() {
 
         return root;
     }
+
+    @Override
+    public String toString() {
+        return MoreObjects.toStringHelper(this)
+                .add("level", level)
+                .add("loggers", loggers)
+                .add("appenders", appenders)
+                .add("loggerContext", loggerContext)
+                .add("configurationErrorsStream", configurationErrorsStream)
```

Comment:
You're right, perhaps I should've written them myself, instead of using the code my IDE blurped out. I'll amend the commit.